src/components/modals/signupModals/signupCollaborator1.tsx

```diff
@@ -20,16 +22,18 @@ export default function SignupModalCollaborator1() {
   const {
     register,
     handleSubmit,
-    setValue,
-    watch,
     formState: { errors },
   } = useForm<FormData>({
     resolver: zodResolver(collaboratorSchema),
   });
 
   // I'd like to send this data to the next modal that pops up so we can send all the data to the server at once.
   // Would I send it through the <SignupModalCollaborator2 /> when I click the form's button?
-  const onSubmit: SubmitHandler<FormData> = (data) => console.log(data);
+  const onSubmit: SubmitHandler<FormData> = (data) => {
+    showModal(<SignupModalCollaborator2 data={data} />);
```

Comment:
Looks like you figured this out? If so, don't forget to remove your comment above and console.log below